hoaithi-rk/meogroup-frontend
# Update Angular Components for Signup Process

## Pull Request

### Title
Refactor Signup Component: Add Form Validation and Update Signup Process

### Description
This pull request refactors the Signup component in the Angular application by adding form validation for username, phone, and email fields. It also updates the signup process to improve user authentication and validation.

### Changes
- Added form validation for username, phone, and email fields in the Signup component.
- Updated the signup process by removing unnecessary fields and updating the signup method in `authen.service.ts`.
- Added new methods and form control initialization in the TypeScript file.
- Added a new color variable `--primary-90` in the CSS file for styling.

### Testing
1. Navigate to the Signup component in the Angular application.
2. Verify that the form fields for username, phone, and email have validation functionality.
3. Enter invalid data in each field and verify that appropriate error messages are displayed.
4. Enter valid data in each field and ensure that the form submission works correctly.
5. Check that the updated signup process functions as expected.

### Related Issues
N/A

### Checklist
- [x] Form validation for username, phone, and email fields added.
- [x] Signup process updated in `authen.service.ts`.
- [x] New methods and form control initialization added in the TypeScript file.
- [x] New color variable `--primary-90` added in the CSS file.
